Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about scuba diving?"
+    m 3eub "after immersion in water, you can see a wonderful underwater world full of diverse marine life."
+    m 7eub "There you will meet:schools of fish, coral reefs, and sea greens,jellyfish's."
+    m 4eub "There may also be sharks."
+    m 2eud "As a rule, they are not in special places for diving."
+    m 7eud "And if there is,then be careful."
+    m 2euc "Although the chance of shark attack is very small."
+    m 3esb "But if you meet a shark, here is one rule for you."
+    m 3esb "Stay calm."
+    m 2euc "Although meeting a shark can be dangerous for you."
```

Comment:
I think it generally goes without saying that shark confrontations can be dangerous, but I think it's a bit out of place to say this, and nothing else. 

Instead, I might combine this with the next line and say "Although coming face to face with a shark can be scary, they usually approach people out of curiosity."